meta/meta.go

```diff
@@ -59,26 +59,25 @@ var (
 //
 
 var (
-	mMetaPrefix              = []byte("m")
-	mNextGlobalIDKey         = []byte("NextGlobalID")
-	mSchemaVersionKey        = []byte("SchemaVersionKey")
-	mDBs                     = []byte("DBs")
-	mDBPrefix                = "DB"
-	mTablePrefix             = "Table"
-	mSequencePrefix          = "SID"
-	mSeqCyclePrefix          = "SequenceCycle"
-	mTableIDPrefix           = "TID"
-	mIncIDPrefix             = "IID"
-	mRandomIDPrefix          = "TARID"
-	mBootstrapKey            = []byte("BootstrapKey")
-	mSchemaDiffPrefix        = "Diff"
-	mPolicies                = []byte("Policies")
-	mPolicyPrefix            = "Policy"
-	mPolicyGlobalID          = []byte("PolicyGlobalID")
-	mPolicyMagicByte         = CurrentMagicByteVer
-	mDDLTableVersion         = []byte("DDLTableVersion")
-	mConcurrentDDL           = []byte("concurrentDDL")
-	mFlashbackHistoryTSRange = []byte("FlashbackHistoryTSRange")
```

Comment:
It's useless, because in two phases, we insert new values instead of delete old values.